frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -101,7 +89,11 @@ export class RhvImportWizard extends ImportWizard {
     // Clicking `edit` button to reach network and storage settings
     await click(rhvView.editButton);
     await click(view.nextButton);
-    await this.next();
+    await click(view.nextButton);
+    await click(view.nextButton);
+  }
+
+  async importConfigStep(name: string, description: string) {
```

Comment:
*General step

Reply:
Renamed it to `importVmConfigStep`, I guess this will better reflect what is going on in this part